patches/extensions-browser-sandboxed_unpacker.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/extensions/browser/sandboxed_unpacker.cc b/extensions/browser/sandboxed_unpacker.cc
+index 5e60ac96251f22aa8b380ea573827b50513b82cd..6152b84d8864bad64e061e4ffab2e205dba019f0 100644
+--- a/extensions/browser/sandboxed_unpacker.cc
++++ b/extensions/browser/sandboxed_unpacker.cc
+@@ -967,6 +967,7 @@ base::DictionaryValue* SandboxedUnpacker::RewriteManifestFile(
+   DCHECK(!public_key_.empty());
+   std::unique_ptr<base::DictionaryValue> final_manifest =
+       manifest.CreateDeepCopy();
++  brave::BraveRewriteManifest(extension_id_, final_manifest.get());
```

Comment:
Using `#define` in an override was not possible here because RewriteManifest is used a couple times in the same file.
Also a subclass would have had the same amount of patched lines and would have complicated things more. So a direct patch was needed as far as I could tell.

Reply:
can we subclass SandboxedUnpacker?

Reply:
We could but RewriteManifest is private so we'd have to patch for that too.  And it's used by 5 other files which create instances.  So I think this 1 line patch is more simple in this case.